quizcon/main/views.py

```diff
@@ -558,9 +557,11 @@ def get_context_data(self, **kwargs):
         quiz_id = self.kwargs.get('pk')
         quiz = get_object_or_404(Quiz, pk=quiz_id)
         submissions = []
-        count = quiz.quizsubmission_set.count()
+        count = 0
         for sub in quiz.quizsubmission_set.all():
```

Comment:
Could you filter this like so rather than doing the loop?

`submissions = quiz.quizsubmission_set.filter(submitted=True)`
then
`total_submissions: submissions.count()`

Reply:
@sdreher I don't think I can filter because submitted isn't a model field. Just a method I created.

Reply:
Ah okay.